content/work/adobe.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Adobe Fonts
+banner: Adobe Fonts
+sub: Adobe Fonts typography superpower game
+logo: adobe
+links:
+  site: https://game.fonts.adobe.com/
+image:
+  src: projects/adobegame.jpg
+  alt: Illustrated game question about which font is most calligraphic
+client: &client Adobe Fonts
+date: 2020-10-20
+people:
+  - &meghan
+    name: Meghan Arnold
+    title: Sr. Product Marketing Manager
+    face: meghan-arnold.jpg
+    url: https://twitter.com/MeghanCArnold/status/1318671589618061312
+    venue: *client
+tasks:
+  - User Experience Design
+  - Graphic Design
+  - Illustration
+  - Animation
+  - Front-End Development
+press:
+  - text: |
+      This was such a fun project to work on 
+      **with @TypeTasting, @OddBird & my @AdobeFonts** colleagues.
```

Comment:
This feels like a strange line to highlight - because it's already clear who we worked with. Why not bold "This was such a fun project to work on"?